packages/google_maps_flutter/google_maps_flutter/pubspec.yaml

```diff
@@ -2,7 +2,7 @@ name: google_maps_flutter
 description: A Flutter plugin for integrating Google Maps in iOS and Android applications.
 repository: https://github.com/flutter/packages/tree/main/packages/google_maps_flutter/google_maps_flutter
 issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3A%22p%3A+maps%22
-version: 2.12.1
+version: 2.12.2
```

Comment:
As indicated in the PR checklist, we follow semver, which requires changing the minor version when adding public API.

Reply:
done.